app/views/channels.html

```diff
@@ -52,10 +53,35 @@
                     <span ng-if="!channel.priority">-</span>
                   </td>
                   <td data-title="Channel Name">
-                    <a ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
+                    <a tooltip={{channel.description}} ng-if="channel.status!='deleted'" href="" ng-click="editChannel(channel)">{{ channel.name }}</a>
                     <span ng-if="channel.status=='deleted'">{{ channel.name }}</span>
                   </td>
                   <td data-title="Channel URL Pattern">{{ channel.urlPattern }}</td>
+                  
+                  <td data-title="Access">
+                    <div class="channelAccessControlDiv" ng-if="channel.status!='deleted'" href="">
+                      
+                      <span ng-if="channel.allow" tooltip="Clients and Roles" ng-click="editChannel(channel, 'Request Matching')">
+                        <i class="glyphicon glyphicon-th"></i>
+                        <a ng-repeat="allow in channel.allow"> {{ allow }}{{$last ? '' : ', '}}</a>
+                      </span>
+                      
+                      <span ng-if="channel.whitelist" tooltip="Whitelisted ip addresses" ng-click="editChannel(channel, 'Request Matching')">
+                        <i class="glyphicon glyphicon-ok"></i>
+                        <a ng-repeat="whitelist in channel.whitelist"> {{ whitelist }}{{$last ? '' : ', '}}</a>
+                      </span>
+                      
+                      <span ng-if="channel.authType=='public'" tooltip="Channel is publically available" ng-click="editChannel(channel, 'Request Matching')">
+                        <i class="glyphicon glyphicon-cloud-upload"></i>
+                        <a > {{ channel.authType }}{{$last ? '' : ', '}}</a>
```

Comment:
Wait, actually, you don't need any commas here. `authType` is just a single string.